css/style.css

```diff
@@ -0,0 +1,270 @@
+b,
+u,
+i,
+center,
+dl,
+dt,
+dd,
+ol,
+ul,
+li,
+fieldset,
+form,
+label,
+legend,
+table,
+caption,
+tbody,
+tfoot,
+thead,
+tr,
+th,
+td,
+article,
+aside,
+canvas,
+details,
+embed,
+figure,
+figcaption,
+footer,
+header,
+hgroup,
+menu,
+nav,
+output,
+ruby,
+section,
+summary,
+time,
+mark,
+audio,
+video {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  font-size: 100%;
+  font-style: inherit;
+  font-variant: inherit;
+  font-weight: inherit;
+  line-height: inherit;
+  font-family: inherit;
+  vertical-align: baseline;
+}
+
+/* HTML5 display-role reset for older browsers */
+article,
+aside,
+details,
+figcaption,
+figure,
+footer,
+header,
+hgroup,
+menu,
+nav,
+section {
+  display: block;
+}
+
+body {
+  line-height: 1;
+}
+
+ol,
+ul {
+  list-style: none;
+}
+
+blockquote,
+q {
+  quotes: none;
+}
+
+blockquote::before,
+blockquote::after,
+q::before,
+q::after {
+  content: '';
+  content: none;
+}
+
+table {
+  border-collapse: collapse;
+  border-spacing: 0;
+}
+
+/*
+  ========================================
+  General CSS
+  ========================================
+*/
+html {
+  font-family: 'Poppins', sans-serif;
+}
+
+h1 {
+  font-size: 40px;
+  font-weight: 700;
+  line-height: 52px;
+}
+
+p {
+  font-size: 16px;
+  font-weight: 400;
+  line-height: 24px;
+}
+
+/*
+  ========================================
+  Reusable font styling classes
+  ========================================
+*/
+.p-medium {
+  font-weight: 500;
+}
+
+.blue-main {
+  color: #172b4d;
+}
+
+.blue-light {
+  color: #344563;
+}
+
+.purple-main {
+  color: #6070ff;
+}
+
+.purple-light {
+  color: #7f8cff;
+}
+
+/*
+  ========================================
+  Reusable Flexbox classes
+  ========================================
+*/
+.flex-row {
+  display: -webkit-box;
+  display: flex;
+  -webkit-box-orient: horizontal;
+  -webkit-box-direction: normal;
+  -ms-flex-direction: row;
+  flex-direction: row;
+}
+
+.flex-col {
+  display: -webkit-box;
+  display: flex;
+  -webkit-box-orient: vertical;
+  -webkit-box-direction: normal;
+  -ms-flex-direction: column;
+  flex-direction: column;
+}
+
+.justify-between {
+  -webkit-box-pack: justify;
+  -ms-flex-pack: justify;
+  justify-content: space-between;
+}
+
+.align-center {
+  -webkit-box-align: center;
+  -ms-flex-align: center;
+  align-items: center;
+}
+
+/*
+  ========================================
+  Navigation bar
+  ========================================
+*/
+nav {
+  padding: 12px 35px 12px 24px;
+}
+
+nav img {
+  vertical-align: middle;
+}
+
+nav button {
+  border: none;
+  background-color: transparent;
+  padding: 0;
+}
+
+nav button:hover {
+  cursor: pointer;
+}
+
+/*
+  ========================================
+  Header
+  ========================================
+*/
+header {
+  padding: 114px 24px;
+  background-image: url("../images/header-background.png");
+  background-repeat: no-repeat;
+}
+
+header > :nth-child(-n+3) {
+  padding-bottom: 12px;
+}
+
+header #social-icons {
```

Comment:
Please avoid using id as a selector. if you attach a style to an ID, you won't be able to reuse it within the same webpage. Classes, however, can appear on several HTML elements on the same page. Being able to reuse styles is one of the advantages of CSS.

Reply:
The ID "social-icons" has now been removed and the class "social" is used instead.

Reply:
:+1: This issue has been addressed successfully :heavy_check_mark: 

=> But as I mentioned earlier, you can checkout the `#headline` id as well and address it